src/ks-app/src/app/containers/wizard/wizards.component.html

```diff
@@ -6,115 +6,181 @@
-                        </span>
-                            </label>
-                        </div>
-                    </section>
-                </form>
-            </clr-wizard-page>
-
-            <clr-wizard-page [clrWizardPageNextDisabled]="!formPageOne.valid || !formPageTwo.valid">
-                <ng-template clrPageTitle>Title for page 3</ng-template> <!-- mandatory -->
-                <ng-template clrPageNavTitle>
-            <span *ngIf="formPageOne.valid && formPageTwo.valid">
-                Ready to go!
-            </span>
-                    <span *ngIf="!formPageOne.valid || !formPageTwo.valid">
-                Not ready yet
-            </span>
-                </ng-template> <!-- optional -->
-
-                <div *ngIf="formPageOne.valid && formPageTwo.valid">
-                    <p>Congratulations! You are done with this wizard.</p>
-                    <form class="compact">
-                        <label>Your information</label>
-                        <section class="form-block">
-                            <div class="form-group">
-                                <label>Your name:</label>
-                                <span>{{ formModel.name }}</span>
-                            </div>
-                            <div class="form-group">
-                                <label>Your favorite food:</label>
-                                <span>{{ formModel.favorite }}</span>
-                            </div>
-                            <div class="form-group">
-                                <label>Your lucky number:</label>
-                                <span>{{ formModel.number }}</span>
-                            </div>
-                        </section>
-                    </form>
-                </div>
-
-                <div *ngIf="!formPageOne.valid || !formPageTwo.valid">
-                    <p>Not quite there yet.</p>
-                </div>
-
-            </clr-wizard-page>
-        </clr-wizard>
-    </div>
+  <div class="clr-col-12">
+    <h4>Async Form validation</h4>
+    <button class="btn btn-primary" (click)="asyncFormOpen = !asyncFormOpen">Open Wizard</button>
+    <clr-wizard #asyncFormWizard [(clrWizardOpen)]="asyncFormOpen" (clrWizardOnFinish)="onFinish()">
+      <clr-wizard-title>Async validation</clr-wizard-title>
+
+      <clr-wizard-button [type]="'cancel'">Cancel</clr-wizard-button>
+      <clr-wizard-button [type]="'previous'">Back</clr-wizard-button>
+      <clr-wizard-button [type]="'next'">Next</clr-wizard-button>
+      <clr-wizard-button [type]="'finish'">Finish</clr-wizard-button>
+
+      <clr-wizard-page
+        clrWizardPagePreventDefault="true"
+        (clrWizardPageOnCommit)="onCommit()"
+        (clrWizardPageOnCancel)="doCancel()"
+      >
+        <ng-template clrPageTitle>Form with async validation</ng-template>
+        <!-- mandatory -->
+
+        <clr-spinner *ngIf="loadingFlag">
+          Loading...
```

Comment:
Same thing here with ellipsis, in fact just put something more specific like `Validating your submission`.